Content.Server/Objectives/ObjectivesSystem.cs

```diff
@@ -83,7 +83,10 @@ private void OnRoundEndText(RoundEndTextAppendEvent ev)
             }
 
             var result = Loc.GetString("objectives-round-end-result", ("count", total), ("agent", agent));
-            result += "\n" + Loc.GetString("objectives-round-end-result-in-custody", ("count", total), ("custody", totalInCustody), ("agent", agent));
+            if (agent == Loc.GetString("traitor-round-end-agent-name"))
```

Comment:
yes its been merged but would you not just add ShowCustody to the event and have traitor set it to true